tests/unit/test_conf.py

```diff
@@ -35,6 +37,7 @@
 )
 from neo4j.api import (
     TRUST_SYSTEM_CA_SIGNED_CERTIFICATES,
+    TRUST_ALL_CERTIFICATES,
```

Comment:
```suggestion
    TRUST_ALL_CERTIFICATES,
    TRUST_SYSTEM_CA_SIGNED_CERTIFICATES,
```
Please sort imports alphabetically.